dotnetv3/SQS/SQSSendMessagesExample/SQSSendMessagesExample/SQSSendMessagesExample.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+/// <summary>
+/// Shows how to send messages to an existing Amazon SQS queue.

```

Comment:
Same comment as before. First reference should be: "Amazon Simple Queue Service (Amazon SQS)" and subsequent mentions should be: "Amazon SQS"